src/Tokenizer/TokensAnalyzer.php

```diff
@@ -64,7 +64,7 @@ public function getClassyElements()
      *
      * @param bool $perNamespace Return namespace uses per namespace
      *
-     * @return array|array[]
+     * @return int[]|int[][]
```

Comment:
the `int[]` is really an empty `array` here I think, than again `int[]` might be a better fit